recipe/build.sh

```diff
@@ -1,16 +1,10 @@
 #!/bin/bash
 
-# FIXME: This is a hack to make sure the environment is activated.
-# The reason this is required is due to the conda-build issue
-# mentioned below.
-#
-# https://github.com/conda/conda-build/issues/910
-#
-source activate "${CONDA_DEFAULT_ENV}"
+# rm -rf $PREFIX/lib/*.la
 
 if [ $(uname) == Darwin ]; then
+    export LDFLAGS="-headerpad_max_install_names"
     OPTS="--enable-rpath"
-    export CXX="${CXX} -stdlib=libc++"
```

Comment:
@gillins if the PR to create a `libgfortran` a is merged we won't need this nor the `toolchain` b/c there is no other source for `libstdc++` on the system (`gcc` from `homebrew` is already removed at the beginning of he CI run).

Let's wait for that PR, but you can try https://github.com/conda-forge/gdal-feedstock/pull/90/files#r70559805 in PR #87 and, if that works, we can merge it to finally get a `gdal` that works.